server/docs/design/bidi-producer-consumers-streaming.md

```diff
@@ -0,0 +1,339 @@
+# Bi-directional Producer/Consumer Streaming with gRPC
+
+## Purpose
+
+A primary use case of the `hedera-block-node` is to stream live BlockItems (see Terms section) from a producer 
+(e.g. Consensus Node) to N consumers (e.g. Mirror Node) with the lowest possible latency while correctly preserving the 
+order of the BlockItems.  This document outlines several possible strategies to implement this use case and the design 
```

Comment:
nit: one space is preferred (https://www.grammarly.com/blog/spaces-after-period/). I know, it is unimportant bike-shedding, but I must stand on my soap box every once in a while :-D

Reply:
Is this something the gradle plugins can detect and handle?  Our repo isn't quite up to speed yet on leveraging the plugin capabilities.

Reply:
Build plugins won't adjust or modify comments or markdown.
English grammar and typesetting rules are pretty much up to us to follow.

Reply:
Bummer.  Ok, I changed all the occurrences in this design doc manually.